qgis-app/styles/views.py

```diff
@@ -26,25 +30,105 @@
 from styles.file_handler import read_xml_style
 
 
-def check_styles_access(user, style):
+def is_style_manager(user: User) -> bool:
+    """Check if user is the members of Style Managers group."""
+
+    return user.groups.filter(name="Style Managers").exists()
+
+
+def check_styles_access(user: User, style: Style) -> bool:
+    """Check if user is the creator of the style or is_staff."""
+
+    return user.is_staff or style.creator == user or is_style_manager(user)
+
+
+def send_mail_wrapper(subject,
+                      message,
+                      mail_from,
+                      recipients,
+                      fail_silently=True):
     """
-    Check if user is the creator of the style or is_staff
-
-    Parameters
-    ----------
-    user : User instance
-        The user whom will be checked against the style
-    style : Style instance
-
-    Returns
-    -------
-    bool
-        Return True if user is_staff or user is style's creator
+    Wrapping send_email function to send email only when not DEBUG.
     """
 
-    if user.is_staff or style.creator == user:
-        return True
-    return False
+    if settings.DEBUG:
+        logging.debug("Mail not sent (DEBUG=True)")
+    else:
+        send_mail(subject,
+                  message,
+                  mail_from,
+                  recipients,
+                  fail_silently)
+
+
+def style_notify(style: Style, created=True) -> None:
+    """
+    Email notification when a new style created.
+
+    """
+    recipients = [u.email for u in User.objects.filter(
+        groups__name="Style Managers").exclude(email='')]
+
+    if created:
+        style_status = "created"
+    else:
+        style_status = "updated"
+
+    if recipients:
+        domain = Site.objects.get_current().domain
+        mail_from = settings.DEFAULT_FROM_EMAIL
+
+        send_mail_wrapper(
+            _('A new style has been %s by %s.') % (style_status,
+                                                   style.creator),
+            _('\r\nStyle name is: %s\r\nStyle description is: %s\r\n'
+              'Link: http://%s%s\r\n') % (style.name, style.description,
+                                          domain, style.get_absolute_url()),
+            mail_from,
+            recipients,
+            fail_silently=True)
+        logging.debug('Sending email notification for %s style, '
+                      'recipients:  %s' % (style.name, recipients))
+    else:
+        logging.warning('No recipients found for %s style notification'
+                        % style.name)
+
+
+def style_approval_notify(style: Style, creator: User, staff: User) -> None:
```

Comment:
style_update_notify

Reply:
the style_approval_notify method has been renamed to style_update_notify